### PR TITLE
Gravity+Speed modifier

### PR DESCRIPTION
**Pull request by WhyFaust**

### CODE REVIEW

File: hl_gangs.sp [hunk 1]
```diff
@@ -1660,7 +1666,7 @@ public void SQLCallback_Perks(Database db, DBResultSet results, const char[] err
 		if (!gcv_bDisableSize.BoolValue && gcv_iMaxGangSize.IntValue != 0)
 		{
 			price = gcv_iSizePrice.IntValue + (gcv_iPriceModifier.IntValue * ga_iSize[client]);
-			Format(sDisplayBuffer, sizeof(sDisplayBuffer), "%T [%i/9] [%i %T]", "GangSize", client, ga_iSize[client], price, "Credits", client);
+			Format(sDisplayBuffer, sizeof(sDisplayBuffer), "%T [%i/gcv_iGangSizeMaxUpgrades] [%i %T]", "GangSize", client, ga_iSize[client], price, "Credits", client);
```
Comment (shanapu) [inline hl_gangs.sp]: Shouldn't this be `gcv_iGangSizeMaxUpgrades.IntValue`?

Comment (shanapu) [reply hl_gangs.sp]: Or more precisely
`Format(sDisplayBuffer, sizeof(sDisplayBuffer), "%T [%i/%i] [%i %T]", "GangSize", client, ga_iSize[client], gcv_iGangSizeMaxUpgrades.IntValue, price, "Credits", client);`

**Comment by Headline** [comment]
Does it's default values match that of the old function?

**Comment by WhyFaust** [comment]
It used to be 0,01~0.02 Now 0,03

**Comment by Headline** [comment]
I'm alright with having OnAvailableLR implemented in this plugin for those who have hosties, but we should also keep in mind that some servers may not be running this plugin. So we have to be careful about how we use this native.

On line 2755 if we keep LastRequest(), but then add OnAvailableLR elsewere and have it call LastRequest(). That way LastRequest() is kept and can be used in the PlayerDeath hook if they're not using hosites (A simple LibraryExists check).


Besides that I'm 👍

**Comment by Headline** [comment]
Thank you!

👍

**Pull request merged**